README.md

```diff
@@ -7,8 +7,7 @@ Scala grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 
 References
 
-* [The Scala Language Specification](http://www.scala-lang.org/docu/files/ScalaReference.pdf)
-* [Scala Syntax Summary](https://www.scala-lang.org/files/archive/spec/2.11/13-syntax-summary.html)
```

Comment:
I'm going to assume that when work was starting on this that it was still in 2.11 time. There has been quite a few updates since then, so probably best to update this, especially seeing that Dotty is around the corner.